test/scenarios/bookshop/funcs.test.js

```diff
@@ -276,6 +305,7 @@ describe('Bookshop - Functions', () => {
     })
     test('now', async () => {
       // REVISIT: this test does not really proof much
+      // new adapter error: 500 - sql error
```

Comment:
`now` is not implemented on the database service. Could be added as a proxy for `session_context('$now')`.